docs/en_us/platform_api/source/user/endpoints.rst

```diff
@@ -15,8 +15,24 @@ The following tasks and endpoints are currently supported.
      - Use this endpoint:
    * - :ref:`Get a user's account information <Get and Update the User's
        Account Information>`
-     - GET /api/user/v0/accounts/{username}/[?view=shared]
+     - GET /api/user/v1/accounts/{username}/[?view=shared]
    * - :ref:`Update your account information <Get and Update the User's Account
        Information>`
-     - PATCH /api/user/v0/accounts/{username}/{“key”:”value”} “application
+     - PATCH /api/user/v1/accounts/{username}/{“key”:”value”} “application
        /merge-patch+json”
+   * - :ref:`Get a user's preferences information <Get and Update the User's
+       Preferences Information>`
+     - GET /api/user/v1/preferences/{username}/
+   * - :ref:`Update a user's preferences information <Get and Update the User's
+       Preferences Information>`
+     - PATCH /api/user/v1/preferences/{username}/ with content_type
```

Comment:
Why does this one say "with content_type ..." while the one above says "application ..."? Is this caused by our docstring? I'm not sure that it is necessary to have the content types shown here. We will eventually support more than one content type for some of these endpoints, so I think it makes sense to have to drill in to see that level of detail.